acct-group/i2p/i2p-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for system-wide net-vpn/i2p"
+ACCT_GROUP_ID=999
```

Comment:
I think using 999 is banned for now. You should send a request about ID/GID to gentoo-dev mailing list, if it gets accepted, I'll look into this. There are many examples in the ML how it works lately.